src/main/java/bio/terra/cli/command/shared/BaseCommand.java

```diff
@@ -55,6 +58,7 @@ public Integer call() {
     }
 
     // execute the command
+    logger.debug("terra " + String.join(" ", Main.getArgList()));
```

Comment:
nit: Suggest prefixing this with something e.g. "[COMMAND RUN]" to distinguish it a bit from the other log statements.